index.js

```diff
@@ -20,19 +28,55 @@ AFRAME.registerComponent('simple-link', {
   /**
    * Called once when component is attached. Generally for initial setup.
    */
-  init() {},
+  init() {
+    this.navigate = this.navigate.bind(this);
+    const { el } = this;
+
+    el.setAttribute('geometry', {
+      primitive: 'circle',
+      radius: 1
```

Comment:
Do you think it would make sense to make the radius configurable?

Reply:
Added